src/prefect/cli/deployment.py

```diff
@@ -439,14 +439,18 @@ async def run(
                     },
                 )
             except Exception as exc:
-                exit_with_error(
-                    f"Exception occurred while parsing '{start_time_raw}': {exc!r}"
-                )
+                exit_with_error(f"Failed to parse '{start_time_raw!r}': {exc!s}")
 
         if start_time_parsed is None:
             exit_with_error(f"Unable to parse scheduled start time {start_time_raw!r}.")
 
-        scheduled_start_time = pendulum.instance(start_time_parsed)
+        if start_time_parsed.tzinfo:
+            exit_with_error(
+                f"Timezone not expected in '{start_time_raw!r}'. Timezone is inferred with pendulum.tz.local_timezone()."
```

Comment:
From a UX perspective, we should definitely let them provide a time zone. We may even want to display in that time zone instead of the local timezone if it's explicitly given.

Reply:
It’s looking like allowing users to specify a time to start-at such as “5pm PST” is going to be difficult because there isn't consistent mapping/meaning for those abbreviations. Alternatively, asking users to specify a locality-based tz seems not to be a good UX (“America/New York”). Another option could be requiring an offset like “5pm UTC-5” which is probably better than nothing. What do you think?